pkg/rpcutil/rpc.go

```diff
@@ -31,13 +32,13 @@ import (
 const grpcDialDefaultTimeout = 2 * time.Second
 
 // GetClientConnection returns a grcp.ClientConn that is unsecured
-func GetClientConnection(addr string, port int) (*grpc.ClientConn, error) {
-	return GetClientConnectionWithCreds(addr, port, nil)
+func GetClientConnection(ctx context.Context, addr string, port int) (*grpc.ClientConn, error) {
```

Comment:
Don't forget to update the test file to reflect this change. Legacy tests are currently failing because of this change. The failure is in control_grpc_server_test.go at line 135. It is failing because not enough arguments were provided.